File: src/main/resources/test-programs/IfStatements.java

public class IfStatements {

    public static void value(long a) {
        String value;
        if (a == 6L) {
            value = "a is 6";
        } else if (a == 7L) {
            value = "a is 7";
        } else if (a > 0L) {
            value = "a is positive";
        } else {
            value = "a is negative";
        }
        System.out.println(value);
    }

    public static void main(String[] args) {
        value(6L);
        value(7L);
        value(8L);
        value(10L - 283L);
    }
}

File: src/main/resources/debug/IfStatements.c
#include <stdio.h>

void value(long long a) {
    if (a == 6) {
        puts("a is 6");
    } else if (a == 7) {
        puts("a is 7");
    } else if (a > 0) {
        puts("a is positive");
    } else {
        puts("a is negative");
    }
}

int main() {
    value(6);
    value(7);
    value(8);
    value(10);
    value(-283);
    return 0;
}
